content/client-lib-development-guide/features.textile

```diff
@@ -55,6 +56,35 @@ h2(#test-guidelines). Test guidelines
+* @(RAS2)@ The primary REST host and primary realtime host are determined based on the supplied @ClientOptions@ as follows.
+** @(RAS2a)@ If the @ClientOptions#restHost@ option is set, the specified host is the primary REST host. It is not valid for @ClientOptions#restHost@ to be specified in conjunction with either a @ClientOptions#environment@ or @ClientOptions#zone@ option.
+** @(RAS2b)@ If the @ClientOptions#realHost@ option is set, the specified host is the primary realtime host. It is not valid for @ClientOptions#realHost@ to be specified in conjunction with either a @ClientOptions#environment@ or @ClientOptions#zone@ option.
+** @(RAS2b)@ Otherwise, the primary hosts are determined by first determining values for @zone@ and @environment@ as follows.
+*** @(RAS2b1)@ The value of @zone@ determines the principal domain (PD) of the primary hosts. If the @ClientOptions#zone@ option is set to the value @production@, or is unset, then the @zone@ is @production@. Otherwise, the value of @zone@ is the value of the @ClientOptions#zone@ option.
+**** @(RAS2b1a)@ A @zone@ value of @production@ corresponds to a PD of @ably.io@.
+**** @(RAS2b1b)@ A @zone@ value of @dev@ corresponds to a PD of @ably-dev.net@.
+**** @(RAS2b1c)@ No other @zone@ values are valid.
+*** @(RAS2b2)@ The value of @environment@ determines the subdomain of the primary host(s). If the @ClientOptions#environment@ option is set to the value @production@, or is unset, then the @environment@ is @production@. Otherwise, the value of @environment@ is the value of the @ClientOptions#environment@ option.
+**** @(RAS2b2a)@ For a @zone@ value of @production@, an @environment@ value of @production@ corresponds to @rest@ and @realtime@ subdomains of the PD for the REST and realtime primary hosts respectively. That is, the primary hosts are @rest.ably.io@ and @realtime.ably.io@.
+**** @(RAS2b2b)@ For a @zone@ value of @production@, any other @environment@ value corresponds to subdomains @<environment>-rest@ and @<environment>-realtime@ for REST and realtime hosts respectively. For example, an @environment@ value of @sandbox@ corresponds to primary hosts @sandbox-rest.ably.io@ and @sandbox-realtime.ably.io@.
+**** @(RAS2b2c)@ For any other @zone@ value, both REST and realtime operations use the same primary host, in which the subdomain equals the @environment@.
+** @(RAS2c)@ The resulting value of the primary host is obtained by combining the PD and subdomains. For example, an @environment@ value of @sandbox@ and @zone@ value of @production@ corresponds to an host of @sandbox-rest.ably.io@. An @environment@ value of @loadtest@ and a @zone@ value of @dev@ corresponds to an host of @loadtest-rest.ably-dev.net@.
```

Comment:
> `An @environment@ value of @loadtest@ and a @zone@ value of @dev@ corresponds to an host of @loadtest-rest.ably-dev.net@.`

This contradicts RAS2b2c above, which has that "both REST and realtime operations use the same primary host, in which the subdomain equals the environment", which would imply `loadtest.ably-dev.net` not `loadtest-rest.ably-dev.net`